src/oss-tests/FindSquatsTests.cs

```diff
@@ -33,9 +34,9 @@ public async Task DetectSquats(string packageUrl, bool expectedToHaveSquats)
         }
 
         [DataTestMethod]
-        [DataRow("pkg:npm/%40angular/core", "@engular/core", "@angullar/core")]
+        [DataRow("pkg:npm/angular/core", "engular/core", "angullar/core")]
         [DataRow("pkg:npm/lodash", "odash", "lodah")]
-        [DataRow("pkg:npm/%40babel/runtime", "@abel/runtime", "@bable/runtime")]
+        [DataRow("pkg:npm/babel/runtime", "abel/runtime", "bable/runtime")]
```

Comment:
Any potential of breaking backwards compatibility for users moving from older versions of the library to this update?

Reply:
Did the change to the packageurl library upstream implicitly change how we handle @'s in scopes?

Reply:
Yeah, we drop the @ in packageurl now, it's just stored in the FindPackageSquatsResult as the name now I think.

Reply:
Back compat still works, just added:

`[DataRow("pkg:npm/%40angular/core", "@engular/core", "@angullar/core")]`

works fine

Reply:
LGTM then.